src/input_utils.py

```diff
@@ -149,7 +177,7 @@ def _decode_fn(record):
       begin, size, _ = tf.image.sample_distorted_bounding_box(
           tf.shape(im),
           tf.zeros([0, 0, 4], tf.float32),
-          area_range=(0.05, 1.0),
+          area_range=(0.85, 1.0),
```

Comment:
FYI: for the first version we don't need to worry about bounding box, but you can leave it as is for now.